### PR TITLE
INSTALL.txt: updated details installation from source

### PR DESCRIPTION
**Pull request by sanderjo**
Closes https://github.com/sabnzbd/sabnzbd/issues/1467

### CODE REVIEW

File: INSTALL.txt [hunk 1]
```diff
@@ -52,19 +52,25 @@ Specific guides to install from source are available for Windows and macOS:
...
     https://sabnzbd.org/wiki/installation/install-from-source-windows
 
-Only Python 3.5 and above is supported. Replace python3 with python in the commands
-below if Python 3 is the only version of Python on your system.
+Only Python 3.5 and above is supported.
 
-Install all required Python packages by running:
-    python3 -m pip install -r requirements.txt
+You need to install
+    par2 unrar unzip python3-setuptools python3-pip
```
Comment (Safihre) [inline INSTALL.txt:58]: This is only for non-Windows/macOS systems:
`On Linux systems you need to install:`

File: INSTALL.txt [hunk 2]
```diff
@@ -52,19 +52,25 @@ Specific guides to install from source are available for Windows and macOS:
...
+
+On non-X86 platforms (for which pypi does not provide all pre-compiled packages),
```
Comment (Safihre) [inline INSTALL.txt:60]: `PyPI`

Comment (Safihre) [reply INSTALL.txt:60]: And maybe remove the `()` in this line.

File: INSTALL.txt (hunk 1 above)
Comment (sanderjo) [reply INSTALL.txt:58]: We're talking here about "4) INSTALL with only sources", not installing from the packages. So, on Windows and MacOS ... don't you need to install unrar etc?

Comment (Safihre) [reply INSTALL.txt:58]: No, these are included here in the repo and when you install Python on Windows it already includes setuptools and pip.

**Pull request merged**